### PR TITLE
EQMS-1033 Filter out readonly projects when creating new controlled document

### PR DESCRIPTION
**Pull request by aonnikov**
<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2Njc5OTUxYjNlOGFhOTFjZTY2YWViOGYiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.Y1Eptu1WV1hShu7PONmgRohbIrVmiVuYnGKRNK8PjBo">Huly&reg;: <b>UBERF-7392</b></a></sub>

### CODE REVIEW

File: plugins/controlled-documents-resources/src/utils.ts [hunk 1]
```diff
@@ -714,7 +721,7 @@ export async function getLatestProjectId (spaceRef: Ref<DocumentSpace>): Promise
 
   if (space !== undefined) {
     if (space.$lookup?.type?.projects ?? false) {
-      const project = await getLatestProject(spaceRef)
+      const project = await getLatestProject(spaceRef, editableOnly)
```
Comment (lexiv0re) [inline plugins/controlled-documents-resources/src/utils.ts]: Should condition be inverted?

Comment (aonnikov) [reply plugins/controlled-documents-resources/src/utils.ts]: no, that's just wrong name :)

**Review by BykhovDenis — APPROVED** [review]

File: plugins/controlled-documents-resources/src/utils.ts (hunk 1 above)
Comment (lexiv0re) [reply plugins/controlled-documents-resources/src/utils.ts]: Ok, but why do we include readonly in document space presenter then? I thought we wanted to only show not readonly versions there? Or it's in another component?

Comment (lexiv0re) [reply plugins/controlled-documents-resources/src/utils.ts]: discussed

**Review by lexiv0re — APPROVED** [review]

**Pull request merged**